init/spark/spark-defaults.conf

```diff
@@ -18,14 +18,14 @@
 #
 
 spark.plugins org.apache.gravitino.spark.connector.plugin.GravitinoSparkPlugin
-spark.sql.gravitino.uri http://gravitino:8090
+spark.sql.gravitino.uri http://__GRAVITINO_HOST_IP__:8090
 spark.sql.gravitino.metalake metalake_demo
 spark.sql.gravitino.enableIcebergSupport true 
 spark.sql.extensions org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions
 spark.sql.catalog.catalog_rest org.apache.iceberg.spark.SparkCatalog
 spark.sql.catalog.catalog_rest.type rest
-spark.sql.catalog.catalog_rest.uri http://gravitino:9001/iceberg/
+spark.sql.catalog.catalog_rest.uri http://__GRAVITINO_HOST_IP__:9001/iceberg/
 spark.locality.wait.node 0
-spark.sql.warehouse.dir hdfs://hive:9000/user/hive/warehouse
-spark.sql.catalog.catalog_hive.spark.sql.hive.metastore.jars path
```

Comment:
@unknowntpo,  any reason to change these two lines?
```
spark.sql.warehouse.dir hdfs://hive:9000/user/hive/warehouse
spark.sql.catalog.catalog_hive.spark.sql.hive.metastore.jars path
```

Reply:
@FANNG1 I think these two line were accidentally changed by me and we should revert it.

```
spark.sql.catalog.catalog_hive.spark.sql.hive.metastore.jars path

...

spark.sql.catalog.catalog_hive.spark.sql.hive.metastore.jars.path file:///opt/spark/jars/*
```

And this line should be reverted since we no longer support helm chart.

```
spark.sql.warehouse.dir
```